core/src/main/java/nu/yona/server/ThreadScope.java

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Stichting Yona Foundation
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v.2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server;
+
+import org.springframework.context.annotation.Scope;
+
+@Scope("thread")
+public @interface ThreadScope
```

Comment:
This interface should be a bit better documented.

Reply:
Agreed. Added a comment.